tests/integration/validatorTests.py

```diff
@@ -39,55 +40,48 @@ def setUpClass(cls):
         column1.required = True
         column1.field_type = stringType
         column1.file_id = 1
-        cls.interfaces.validationDb.populateFile(column1)
```

Comment:
Looks like the tests are still passing. Pinging @jworcestBAH to ensure we're not overlooking an important use of that method.